### PR TITLE
editorial changes to Linux pages

### PR DESCRIPTION
**Pull request by ChaseCrum**
Made editorial changes for Linux based pages

### CODE REVIEW

**Review by dbradish-microsoft — APPROVED** [review]
@Jiasli, I am fine with the gramatical changes.  Please review for technical accuracy.

File: docs-ref-conceptual/includes/cli-install-linux-dnf.md [hunk 1]
```diff
@@ -12,15 +12,17 @@ ms.custom: devx-track-azurecli
 For Linux distributions with `dnf` such as RHEL 8/9 or CentOS Stream 8/9, there are RPM packages for Azure CLI.
 
 > [!WARNING]
-> Starting from Azure CLI 2.39.0 (released on 08/02/2022), no RPM packages will be released for RHEL 7 and CentOS 7. You may continue to use historical versions of Azure CLI on these systems, but there will be no updates or bugfixes. Consider upgrading to newer versions of RHEL or CentOS to use the latest Azure CLI.
+> Starting from Azure CLI 2.39.0 (released on 08/02/2022), no RPM packages will be released for RHEL 7 and CentOS 7. You may continue to use historical versions of Azure CLI on these systems, but there will be no further updates or bug fixes from Microsoft for those versions.
+>
+> Azure CLI and the Linux [Install script](https://learn.microsoft.com/cli/azure/install-azure-cli-linux?pivots=script) will continue to be developed and supported by Microsoft. Packages provided by Red Hat adn CentoS repositories are maintained and supported by their respective distributions.
```
Comment (bebound) [inline docs-ref-conceptual/includes/cli-install-linux-dnf.md:17]:
```suggestion
> Azure CLI and the Linux [Install script](https://learn.microsoft.com/cli/azure/install-azure-cli-linux?pivots=script) will continue to be developed and supported by Microsoft. Packages provided by Red Hat and CentOS repositories are maintained and supported by their respective distributions.
```

**Comment by dbradish-microsoft** [comment]
I am merging this PR with sufficient time left for the Engineering team review.

**Comment by bebound** [comment]
This PR removes `<div id="set-release"/>` and cause this warning
```
Line 99, Column 93: [Warning: bookmark-not-found] Cannot find bookmark '#set-release' in 'docs-ref-conceptual/includes/cli-install-linux-apt.md'.
Line 105, Column 1: [Warning: bookmark-not-found] Cannot find bookmark '#set-release' in 'docs-ref-conceptual/includes/cli-install-linux-apt.md'.
Line 111, Column 54: [Warning: bookmark-not-found] Cannot find bookmark '#set-release' in 'docs-ref-conceptual/includes/cli-install-linux-apt.md'.

```
@ChaseCrum Could you please fix this?

**Pull request merged**